validator/client/src/main/java/tech/pegasys/teku/validator/client/AttestationDutyScheduler.java

```diff
@@ -35,14 +39,35 @@ public AttestationDutyScheduler(
   public void onAttestationCreationDue(final UInt64 slot) {
     // Check slot being null for the edge case of genesis slot (i.e. slot 0)
     if (lastAttestationCreationSlot != null && slot.compareTo(lastAttestationCreationSlot) <= 0) {
+      LOG.debug(
+          "lastAttestationCreationSlot {} is ahead of current slot {}, not processing attestation duty.",
+          lastAttestationCreationSlot,
+          slot);
       return;
     }
+
+    if (!isAbleToVerifyEpoch(slot)) {
+      LOG.info(
+          "current epoch {} could not be verified against the current slot {}, not processing attestation duty",
+          getCurrentEpoch().map(UInt64::toString).orElse("UNDEFINED"),
+          slot);
```

Comment:
```suggestion
      LOG.info(
          "Not performing attestation duties for slot {} because it is too far ahead of the current slot {}",
          slot,
          getCurrentEpoch().map(UInt64::toString).orElse("UNDEFINED"));
```